src/index.tsx

```diff
@@ -226,6 +226,56 @@ export class App extends React.Component<{}, AppState> {
     } else if (event.metaKey && event.shiftKey && event.code === "KeyV") {
       this.pasteStyles();
       event.preventDefault();
+
+      // Ungroup elements: Cmd-Shift-G
+    } else if (event.metaKey && event.shiftKey && event.code === "KeyG") {
+      this.ungroupElements();
+      this.forceUpdate();
+      event.preventDefault();
+
+      // Group elements: Cmd-G
+    } else if (event.metaKey && event.code === "KeyG") {
```

Comment:
can you please replace the `event.metaKey` with `event[META_KEY]` so that it works on Windows? I'm not sure if you gave collaborators permissions to edit the PR themselves, so not sure if I can do it from my end.